common/source/docs/common-network.rst

```diff
@@ -0,0 +1,141 @@
+.. _common-network:
+
+===========================
+ArduPilot Network Interface
+===========================
+
+As of firmware 4.5 and later, ArduPilot provides a network interface framework to allow local and wide-area network connections. Several later generation autopilots, like Pixhawk6X and CubePilot CubeRed, provide Ethernet physical layer interfaces, which allows ArduPilot to connect to vehicle peripherals, data servers, and even the wide area network via IP using transport layer UDP or TCP protocols. In addition, custom builds for H7 based processors can be created with the `ArduPilot Custom Firmware Server <https://firmware.ardupilot.org/>`__ that includes PPP(Point-To-Point Protocol) allowing network connections over a serial port of the autopilot using PPP.
```

Comment:
Because it's all one paragraph, this reads like PPP will only work on AP's with an ethernet adapter. Suggest breaking it out into 3 paragraphs, or make the last two sentences bullet points.
As of firmware 4.5 ...

- Several later generation autopilots ...
- In addition ...

Reply:
done